web/styles/popovers.css

```diff
@@ -232,6 +232,13 @@ ul {
     }
 }
 
+.topics_popover {
```

Comment:
This section of CSS appears to be duplicating the code on line 23 of `popovers.css`. In my opinion, it may not be necessary, but you should check this!

Reply:
Removed it, after checking, thanks for finding that out.